.travis.yml

```diff
@@ -65,35 +66,54 @@ jobs:
   include:
     - stage: lint
       python: 3.6
-      env: MOCK= NOSETIMER= NOSE= CODECOV= COVERAGE= NUMPY= CYTHON= MATPLOTLIB= SYMPY= H5PY= SCIPY= FASTCACHE= IPYTHON= PANDAS=
+      env: MOCK= NOSETIMER= NOSE= CODECOV= COVERAGE= NUMPY= CYTHON= MATPLOTLIB= SYMPY= H5PY= SCIPY= FASTCACHE= IPYTHON= PANDAS= REQUESTS=
       script: flake8 yt/
 
     - stage: lint
       python: 2.7
-      env: MOCK= NOSETIMER= NOSE= CODECOV= COVERAGE= NUMPY= CYTHON= MATPLOTLIB= SYMPY= H5PY= SCIPY= FASTCACHE= IPYTHON= PANDAS=
+      env: MOCK= NOSETIMER= NOSE= CODECOV= COVERAGE= NUMPY= CYTHON= MATPLOTLIB= SYMPY= H5PY= SCIPY= FASTCACHE= IPYTHON= PANDAS= REQUESTS=
       script: flake8 yt/
 
     - stage: tests
+      name: "Python: 2.7 Unit Test"
       python: 2.7
-      env: NUMPY=numpy==1.10.4 CYTHON=cython==0.24 MATPLOTLIB=matplotlib==1.5.3 SYMPY=sympy==1.0 H5PY= SCIPY= FASTCACHE= FLAKE8= IPYTHON=ipython==1.0
-      script: coverage run $(which nosetests) -c nose.cfg yt
+      env: NUMPY=numpy==1.10.4 CYTHON=cython==0.24 MATPLOTLIB=matplotlib==1.5.3 SYMPY=sympy==1.0 H5PY= SCIPY= FASTCACHE= FLAKE8= IPYTHON=ipython==1.0 REQUESTS=
+      script: coverage run $(which nosetests) --attr "!answer_test" -c nose_unit.cfg
 
     - stage: tests
+      name: "Python: 2.7 Unit Test"
       python: 2.7
-      env: FLAKE8=
-      script: coverage run $(which nosetests) -c nose.cfg yt
+      env: FLAKE8= REQUESTS=
+      script: coverage run $(which nosetests) --attr "!answer_test" -c nose_unit.cfg
 
     - stage: tests
+      name: "Python: 3.5 Unit Test"
       python: 3.5
-      env: FLAKE8=
-      script: coverage run $(which nosetests) -c nose.cfg yt
+      env: FLAKE8= REQUESTS=
+      script: coverage run $(which nosetests) --attr "!answer_test" -c nose_unit.cfg
+
+    - stage: tests
+      name: "Python: 3.6 Unit Test"
+      python: 3.6
+      env: FLAKE8= REQUESTS=
+      script: coverage run $(which nosetests) --attr "!answer_test" -c nose_unit.cfg
+
+    - stage: tests
+      name: "Python: 2.7 Answer Test"
+      python: 2.7
+      env: NUMPY=numpy==1.10.4 CYTHON=cython==0.24 MATPLOTLIB=matplotlib==1.5.3 SYMPY=sympy==1.0 H5PY= SCIPY= FASTCACHE= FLAKE8= IPYTHON=ipython==1.0
+      script: coverage run $(which nosetests) --attr "answer_test" --xunit-file ANS_NOSETEST_XML -c nose_answer.cfg
+      after_failure: python tests/report_failed_answers.py -f -m --xunit-file ANS_NOSETEST_XML
 
     - stage: tests
+      name: "Python: 3.6 Answer Test"
       python: 3.6
       env: FLAKE8=
-      script: coverage run $(which nosetests) -c nose.cfg yt
+      script: coverage run $(which nosetests) --attr "answer_test" --xunit-file ANS_NOSETEST_XML -c nose_answer.cfg
+      after_failure: python tests/report_failed_answers.py -f -m --xunit-file ANS_NOSETEST_XML
 
     - stage: tests
+      name: "Xcode: xcode7.3 Unit Test"
```

Comment:
This should maybe be `Unit Tests` (with an `s`) and `Answer Tests`.

Reply:
fixed